curriculum/test/test-challenges.js

```diff
@@ -253,6 +251,44 @@ function populateTestsForLang({ lang, challenges, meta }) {
   const challengeTitles = new ChallengeTitles();
   const validateChallenge = challengeSchemaValidator();
 
+  describe('Assert meta order', function () {
+    /** This array can be used to skip a superblock - we'll use this
+     * when we are working on the new project-based curriculum for
+     * a superblock (because keeping those challenges in order is
+     * tricky and needs cleaning up before deploying).
+     */
+    const superBlocksUnderDevelopment = ['responsive-web-design'];
+    const superBlocks = new Set([
+      ...Object.values(meta)
+        .map(el => el.superBlock)
+        .filter(el => !superBlocksUnderDevelopment.includes(el))
+    ]);
+    superBlocks.forEach(superBlock => {
+      const filteredMeta = Object.values(meta)
+        /**
+         * Exclude any meta which doesn't have a superOrder, as these shouldn't
+         * appear on the learn map and thus don't need to be validated.
+         */
+        .filter(el => el.superBlock === superBlock && el.superOrder)
```

Comment:
```suggestion
        .filter(el => el.superBlock === superBlock && typeof el.superOrder !== 'undefined')
```
It's probably fine as is, but I think we'd want something with `superOrder == 0` to be the zeroth element, rather than filtered.

Reply:
Given that the `order` property for blocks is now configured to be `1` indexed, would we not want `superOrder` to also be `1` indexed?

Reply:
I'm sorry, Nick, I didn't think this through.  We have a few cases where we rely on the order and challengeOrder being zero based.  For example:
https://github.com/freeCodeCamp/freeCodeCamp/blob/7a5e8057692b2364263797fbe59cef2f4f4c1627/api-server/src/server/boot/challenge.js#L153-L160

I don't really mind which way we go (1 based or 0 based), but we should be consistent with all three orders.  Otherwise confusion is bound to arise.

Reply:
So `challengeOrder` is based on the index of the array, which is 0-indexed.

Changing that seems counter-intuitive. I can set the others to be 0 based, but it'll be a *lot* of changes.

Reply:
That's correct. Now you mention it, having them all match challengeOrder is *probably* the smartest approach, since its naturally 0-based

I think consistency is worth having, but not necessarily in this PR. If you'd rather, we can track this in an issue and I'll make the changes when I have the time.

Reply:
Alright, pushed a fix to adjust this. Entirely possible I missed something, though it builds locally and looks correct to me.